packit/local_project.py

```diff
@@ -510,5 +510,9 @@ def reset(self, ref: str):
         """ git reset --hard $ref """
         self.git_repo.head.reset(ref, index=True, working_tree=True)
 
+    @property
+    def head_branch(self) -> str:
+        return self.git_repo.head.name
```

Comment:
This may seem obvious what it does but I'd really appreciate an example here. Especially when talk about gitpython.